<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_VBC</class>
 <widget class="QDialog" name="Dialog_VBC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VBC Calibration</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1111</width>
     <height>741</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_image"/>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>770</y>
     <width>751</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Status</string>
   </property>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>LOREM IPSUM</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_status_sub">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>LOREM IPSUM</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_training">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Main">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>820</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_progressbar">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>840</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>750</y>
     <width>1141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_deviation">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>770</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Standard Deviation: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mean">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>790</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mean:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_deviation_text">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>770</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Standard Deviation: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mean_text">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>790</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mean:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
